python/fledge/services/core/server.py

```diff
@@ -1796,6 +1799,12 @@ def get_token_common(cls, request):
             msg = "Authorization header is missing"
             raise web.HTTPBadRequest(reason=msg, body=json.dumps({"error": msg}))
 
+        if not "Bearer " in auth_header:
+            msg = "Invalid Authorization token"
+            # FIXME: raise UNAUTHORISED here and among other places 
+            #   and JSON body to have message key
```

Comment:
Though `error` is a correct key, but as we are using `message` since this started to be in response body, better to be consistent.